README.md

```diff
@@ -1 +1,50 @@
-# forks-and-clones-readme
+# Forks and Clones
+
+## Objectives
+
+1. Fork a repo
+2. Clone a repo
+3. Push to different remote repos
+
+
+##Forking a Github Repository
+Forking a Github repository is just a way to create your own copy of any other Github repo. We do this all the time while using Learn, so you can use your copy as a sandbox to play around while maintaining a canonical repo where the original lab remains intact. In other words, it creates another remote that allows you to work in your own world without having to mess with our learn-co-student stuff.
+
+You can fork any repo by clicking the "Fork" button at the top right any Github repository. 
+
+![](https://github-images.s3.amazonaws.com/enterprise/2.2/assets/images/help/repository/fork_button.jpg)
```

Comment:
don't hotlink to stuff on github, they may change those urls! DL it and upload it to S3 using [this guide](https://github.com/learn-co-curriculum/learn-readme-style-guide#amazon-s3-1)